directembedding/src/main/scala/ch/epfl/directembedding/Utils.scala

```diff
@@ -87,3 +93,16 @@ trait DirectEmbeddingUtils extends MacroModule with TransformationUtils {
   }
 
 }
+
+trait TypeHelper extends MacroModule {
+  import c.universe._
+
+  private def name(s: Symbol): String = s.name.toString.trim
+
+  def equalSymbols(a: Symbol)(b: Symbol): Boolean =
```

Comment:
This has worked for all the tests so far, but I agree there must be a better way.